packages/oui-navbar/src/navbar.html

```diff
@@ -104,8 +104,19 @@ <h3 class="oui-navbar-menu__title" ng-bind="::asideLink.headerTitle || asideLink
                     ng-class="::!!subLink.isActive ? 'oui-navbar-menu__item_active' : null"
                     ng-click="$ctrl.toggleMenu()"
                     ng-href="{{::subLink.url}}"
-                    ng-bind-html="::subLink.template"
                     oui-navbar-group="{{::asideLink.name}}">
+                    <span class="notification__icon">
+                        <i class="oui-icon oui-icon_bicolor"
+                           ng-class=":: 'oui-icon-' + subLink.level + '_circle'"
+                           aria-hidden="true"></i>
+                    </span>
+                    <div>
```

Comment:
I think you should add a classname like `notification__content`, to avoid having a nude `div` in our template